### PR TITLE
Modal Message Component

### PR DESCRIPTION
**Pull request by natalynyu**
This PR adds in the `ModalMessage` component, which will be used to display messages in the Course Planner modals. This component is a span that accepts an optional `id` property, `children` to display the text, and a `variant` as different message types are styled differently. The style variants match those of the `Button` component.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality
      to change)

## Checklist:

- [x] I have run `eslint` on the code
- [x] I have added JSDoc for all of my code (where applicable)
- [x] I have added tests to cover my changes.

## Priority:
- [x] Normal 
- [ ] High 

## Related Issues:

Fixes #373

### CODE REVIEW

File: src/Modals/ModalMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import styled, { ThemeContext } from 'styled-components';
+import { fromTheme, VARIANT } from '../Theme';
+
+export interface ModalMessageProps {
+  /** The id of the modal message */
+  id?: string;
+  /** Text to be displayed */
+  children: string;
+  /** Allows you to pass in a variant property from the VARIANT enum */
+  variant: VARIANT;
```
Comment (rmainseas) [inline src/Modals/ModalMessage.tsx:13]: Should this prop default to `VARIANT.BASE`?

**Review by rmainseas — CHANGES_REQUESTED** [review]
Looks good to me, just one small thing

File: src/Modals/ModalMessage.tsx (hunk 1 above)
Comment (natalynyu) [reply src/Modals/ModalMessage.tsx:13]: @rmainseas Good idea! I added this in 74d7a33.

**Review by rmainseas — APPROVED** [review]
Looks great!

**Pull request merged**